switchStatements/switch-challenge.js

```diff
@@ -0,0 +1,97 @@
+/*## Challenge: Tell me the day!
+
+MVP:
+Create a switch block that will log a string for each day of the week.
+
+Create a variable called day
+The switch will be given a number and return the matching days of the week
+e.g. if day = 2, the switch should print Today is Tuesday in the console.
+Your switch block should also handle numbers out of range */
+
+let number = 23;
+let day = number % 7;
+
+switch (day){
+    case 1: 
+        console.log("Today is Monday.");
+        break;
+    case 2: 
+        console.log("Today is Tuesday.");
+        break;
+    case 3: 
+        console.log("Today is Wednesday.");
+        break;
+    case 4: 
+        console.log("Today is Thursday.");
+        break;
+    case 5: 
+        console.log("Today is Friday.");
+        break;
+    case 6: 
+        console.log("Today is Saturday.");
+        break;
+    case 0: 
+        console.log("Today is Sunday.");
+        break;
+}
+/*## Challenge: Eye Colour
+
+MVP:
+Tell the user what eye colour their child would have.
+
+Create two variables parent1 and parent2
+Given the eye colour of each parent, it should console log the probability of their child having different eye colours, for example: Your eye colours are brown and blue, the chances of your child having blue eyes is --%, brown is --% and green is --%
+Use the following image as a reference */
+
+let parent1 = "blue";
+let parent2 = "green";
+let eyeCombo = parent1 + parent2;
+let childEyeColor;
+
+switch (eyeCombo){
+    case "brownbrown":
+        console.log(`Your eye colours are ${parent1} and ${parent2}, the chances of your child having blue eyes is 6.25%, brown is 75% and green is 18.75%`);
+        childEyeColor = "brown";
+        break;
+    case "greenbrown":
```

Comment:
cool use of switch statements

Reply:
thanks, i just though it would be simpler this way